src/types.ts

```diff
@@ -383,6 +384,7 @@ interface ConfigBase {
   isLiveServer: boolean
   emailMode: EmailOperationMode
   maintenanceMode: boolean
+  Throttle: EventThrottle
```

Comment:
The Throttle used to be only used in one place, but now we need to make the single instance globally available, hence part of config object.